libnetwork/types/network.go

```diff
@@ -289,3 +289,8 @@ type TeardownOptions struct {
 
 // FilterFunc can be passed to NetworkList to filter the networks.
 type FilterFunc func(Network) bool
+
+type NetworkCreateOptions struct {
+	// IgnoreIfExists if true, do not fail if the network already exists
+	IgnoreIfExists bool `json:"ignore_if_exists"`
```

Comment:
I don't think we need to serialize/deserialize this as json in podman, so I would just remove the extra json tag.

Reply:
Done